charts/hedera-json-rpc-relay-websocket/ci/default-values.yaml

```diff
@@ -0,0 +1,9 @@
+# Default values for json-rpc-relay in CI. Configured to testnet
```

Comment:
In a follow up PR we should add a test pod under this charts template to verify it on chart install in CI

Reply:
@isavov making sure you saw this one to add it to any follow up PRs